packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.tsx

```diff
@@ -18,7 +18,8 @@ export class HoverCard extends BaseComponent<IHoverCardProps, IHoverCardState> {
     cardOpenDelay: 500,
     cardDismissDelay: 100,
     expandedCardOpenDelay: 1500,
-    instantOpenOnClick: false
+    instantOpenOnClick: false,
+    defaultFirstFocus: true
```

Comment:
Hmm. I believe the naming here is wrong. I believe others like `TeachingBubble` have `setInitialFocus`.